addons/gamepad/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Gamepad support",
-  "description": "Interact with projects using a controller/gamepad.",
+  "description": "Interact with projects using a USB or Bluetooth controller/gamepad.",
```

Comment:
Chrome only addon?

Reply:
The Gamepad API and Web Bluetooth/USB are different things. All major browsers support the gamepad API https://caniuse.com/?search=gamepad